src/components/AllBeers.js

```diff
@@ -0,0 +1,55 @@
+import Header from "./Header";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Search from "./Search";
+import BeerList from "./BeerList";
+import {Container, Row} from 'react-bootstrap'
+
+function AllBeers() {
+  const [allBeers, setallBeers] = useState([]);
+  const [query, setQuery] = useState("");
+  const baseURL = "https://ih-beers-api2.herokuapp.com/beers";
+
+  useEffect(() => {
+    if (query === "") {
+      axios
+        .get(`${baseURL}`)
+        .then((response) => {
+          setallBeers(response.data);
+        })
+        .catch((e) => {
+          console.log(e);
+        });
+    } else {
+      axios
+        .get(`${baseURL}/search?q=${query}`)
+        .then((response) => {
+          setallBeers(response.data);
+        })
+        .catch((e) => {
+          console.log(e);
+        });
+    }
+  });
```

Comment:
As a rule of thumb never use an `useEffect` without the dependecy array, with that one you could remove the blinking 

```suggestion
  }, [query]);
```